units/Cargo.toml

```diff
@@ -29,3 +29,6 @@ serde = { version = "1.0.103", default-features = false, features = ["derive"],
 [dev-dependencies]
 serde_test = "1.0"
 serde_json = "1.0"
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(kani)'] }
```

Comment:
Just out of interest, why is this level = "warn" and the others are level = "deny"?

Reply:
Sweet, review for the win.

Reply:
Fixed.